eZ/Publish/API/Repository/Tests/_fixtures/Legacy/Sibling.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+return eZ\Publish\API\Repository\Values\Content\Search\SearchResult::__set_state(array(
+   'facets' =>
+  array(
+  ),
+   'searchHits' =>
+  array(
+    0 =>
+    eZ\Publish\API\Repository\Values\Content\Search\SearchHit::__set_state(array(
+       'valueObject' =>
+      array(
+        'id' => 4,
+        'title' => 'Users',
```

Comment:
Hmm, I actually thought that sibling is supposed to work differently.
For the structure:
```
- Home
    +- Something 1
    +- Contact Us (Content id = 58)
    +- Something 2
+- Users
+- Media
```
I expected Sibling for content id = 58 to return `Something 1` and `Something 2`. Parent Location also doesn't add up here since parent of content_id=58 is node_id=2.

Reply:
The result will be exactly as you expect for the given content structure. Check it out. 

There is the following content structure in the test repository:

```
- Root (lid= 1)
   + Home (cid = 57, lid = 2)
   + Media (cid = 41, lid = 43)
   + Setup (cid = 45, lid = 48)
   + Users (cid = 4, lid = 5)
   + Design(cid = 56, lid = 58) <-
```

We are looking for siblings of "Design" and the result is "Home", "Media", "Setup" and "Users".